src/snowflake/snowpark/modin/pandas/io.py

```diff
@@ -514,7 +514,7 @@ def read_excel(
     storage_options: StorageOptions = None,
     dtype_backend: DtypeBackend | NoDefault = no_default,
     engine_kwargs: dict | None = None,
-) -> DataFrame | dict[IntStrT, DataFrame]:  # pragma: no cover
+) -> pd.DataFrame | dict[IntStrT, pd.DataFrame]:  # pragma: no cover
```

Comment:
a quick question, if we do not add pd here, what happens? does it mean no link will be generated for the DataFrame class?

Reply:
This error gets produced when trying to build docs:

```
/Users/joshi/code/snowpandas/src/snowflake/snowpark/modin/pandas/io.py:docstring of snowflake.snowpark.modin.pandas.io.read_excel:: WARNING: more than one target found for cross-reference 'DataFrame': modin.pandas.DataFrame, snowflake.snowpark.modin.pandas.dataframe.DataFrame, snowflake.snowpark.DataFrame, snowflake.snowpark.dataframe.DataFrame
```

After changing it to `pd.DataFrame`, no link is generated for the class, but the warning at least is no longer raised.

Reply:
is it the warning that here are conflict definition of DataFrame, but I thought we have addressed this problem

Reply:
I think sphinx previously couldn't see the `modin.pandas.DataFrame` class, but now it can. This should become unnecessary once we remove dataframe.py.